apps/web/src/routes/_authenticated/dashboard/forensics.lazy.tsx

```
@@ -253,6 +254,34 @@ export default function DashboardForensics() {
+      if (isDashboardAuthError(error)) {
+        markSessionExpired();
+        return;
+      }
+      toast.error('Could not reveal license key', {
+        description: 'You may not have access, or no key is on file for this license.',
+      });
+    },
+  });
+
+  // Revealed keys are scoped to the current lookup result.
+  useEffect(() => {
```

Biome (2.4.16)

[error] This hook specifies more dependencies than necessary: lookupResult.

(lint/correctness/useExhaustiveDependencies)